docs/source/examples/checks/distribution/train_test_feature_drift.ipynb

```diff
@@ -50,8 +50,9 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    %autoreload 2
</pre><p>can we remove this notebook extensions from the docs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/593/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>